src/app/services/project-http.service.ts

```diff
@@ -0,0 +1,25 @@
+import {Injectable} from '@angular/core';
+import {HttpClient} from '@angular/common/http';
+import {BaseHttpService} from "./base-http.service";
+import {Project, ProjectInterface} from "../models/project";
+
+@Injectable()
+export class ProjectHttpService extends BaseHttpService {
```

Comment:
better not to extend service like that. Create CommonHTTPService wich will have exposed request methods (GET, POST etc) and apply base logic there